src/__tests__/companies.test.ts

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line node/no-unpublished-import
+import supertest from 'supertest';
+import app from '../app';
+import companies2018 from './__fixtures__/companies-2018.json';
+
+const request = supertest(app);
+
+describe('GET /api/companies/:year', () => {
```

Comment:
Does these test depend on doing actual requests to the Oslo Børs API when run? If yes, we should instead mock the response from Oslo Børs, so that the tests still pass as green even if Oslo Børs is down (or has changed its API).

If we want to actually check if the scarper is able to scrape the webpage, we should include that as a separate integration test

Reply:
This is kind of the integration tests where the api response scrapes the actual webpage and is compared against a previously known result as JSON. Any suggestions on how to structure the tests?